vllm/worker/model_runner.py

```diff
@@ -127,6 +127,8 @@ def __init__(
         self.flashinfer_workspace_buffer: torch.Tensor
         # Set after load_model.
         self.lora_manager: Optional[LRUCacheWorkerLoRAManager] = None
+        # Register call site for sizing broadcast buffer.
+        self.prepare_input_tensors_callsite_id = register_broadcast_callsite()
```

Comment:
This code is too intrusive. I thought you can extract callsite information automatically, from the function's call stack.

Reply:
@youkaichao yes this is possible but a little hacky imo and there's a non-negligible cost to interrogating the stack (including file access I think).

IMHO this is minimally intrusive, especially since: it's not a public API, it's not used from many places, no other class needs to be updated/kept in sync per call site as is the case with the original approach.

In fact we could just remove this so that we aren't sizing per call site - empirically the latency difference due to the tensor size differences in question here isn't measurable and so could be premature optimization in any case. Especially if we combine the two call sites into one.